codec/decoder/core/src/wels_decoder_thread.cpp

```diff
@@ -57,6 +57,12 @@
 #define HW_NCPU_NAME "hw.ncpu"
 #endif
 #endif
+#ifdef ANDROID_NDK
+#include <cpu-features.h>
+#endif
+#ifdef __ANDROID__
+#include <android/api-level.h>
+#endif
```

Comment:
Why do you readd these includes here? This looks like an accident.

Reviewers, please try to be careful and look for these kind of accidental changes.

This was intentionally removed in #3183, as the include path for cpu-features.h isn't available when building codec/decoder/*, only when building codec/common/*, and there's no need to actually use these functions here. Please see #3192 where I fix building for android again.